usr/src/uts/common/fs/zfs/sys/zil_impl.h

```diff
@@ -142,6 +141,8 @@ typedef struct zil_bp_node {
 
 #define	ZIL_MAX_LOG_DATA (SPA_OLD_MAXBLOCKSIZE - sizeof (zil_chain_t) - \
     sizeof (lr_write_t))
+#define	ZIL_MAX_COPIED_DATA \
+    ((SPA_OLD_MAXBLOCKSIZE - sizeof (zil_chain_t)) / 2 - sizeof (lr_write_t))
 
```

Comment:
That makes sense.  I think a little comment here would have put me on track to figure it out: 
/* Fit at least two log records into each maximum sized block */